.github/workflows/tb-sync.yaml

```
@@ -1,6 +1,10 @@
 on:
   workflow_call:
 
+env:
+  TEMPLATEBUILDER: platformsh/template-builder
+  TBUPDATEBRANCH: updatesLocal
```

```suggestion
  TBUPDATEBRANCH: synced-updates
```
Maybe something a bit more descriptive here?

---

_THIS_ branch name corresponds to the [branch name set in template builder](https://github.com/platformsh/template-builder/blob/master/project/__init__.py#L68) for me to determine if the recent push was from a push from template-builder.